processor/spanprocessor/span.go

```diff
@@ -238,3 +257,32 @@ func (sp *spanProcessor) processToAttributes(span *tracepb.Span) {
 		}
 	}
 }
+
+// skipSpan determines if a span should be processed.
+// True is returned when a span should be skipped.
+// False is returned when a span should not be skipped.
+// The logic determining if a span should be processed is set
+// in the attribute configuration with the include and exclude settings.
+// Include properties are checked before exclude settings are checked.
+func (sp *spanProcessor) skipSpan(span *tracepb.Span, serviceName string) bool {
+	// By default all spans are processed when no include and exclude properties are set.
+	if sp.include == nil && sp.exclude == nil {
+		return false
+	}
```

Comment:
This looks superfluous, same checks are below.

Reply:
removed